edgexfoundry/app-service-configurable
# refactor: Consume MakeItRun rename to Run

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

https://github.com/edgexfoundry/edgex-docs/pull/1060

## Testing Instructions
Run non-secure EdgeX
Build ASC
Run asc with `-cp -d -p sample`
Verify service starts up and is waiting for messages

## New Dependency Instructions (If applicable)
